app/src/main/res/values/strings.xml

```diff
@@ -91,6 +91,10 @@
 
     <!-- App Backup and Restore State -->
 
+    <string name="backup_sms">SMS text messages</string>
```

Comment:
"SMS text messages" one feels somewhat redundant - and this might include MMS too.

Reply:
Google uses the same language though: https://github.com/stevesoltys/seedvault/issues/117#issuecomment-701335659
Also, I think MMS might not always get included as Google's non-free backup as a special class for that.